testing/testcafe/tests/scheduler/grouping/resourceCellTemplate.ts

```diff
@@ -0,0 +1,38 @@
+import { Selector } from 'testcafe';
+import createWidget from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+
+fixture.disablePageReloads`ResourceCellTemplate`
+  .page(url(__dirname, '../../container.html'));
+
+test('resourceCellTemplate layout should be rendered right in the agenda view', async (t) => {
+  const groupHeader = Selector('.dx-scheduler-group-header-content');
```

Comment:
You can apply to selector through test helper. Please, look _testing/testcafe/model/scheduler/headerPanel.ts_